src/main/java/it/pagopa/interop/probing/caller/config/client/RestClientConfig.java

```diff
@@ -18,6 +20,7 @@ public class RestClientConfig {
 
   public RestClientConfig() {
     feignRestClient = Feign.builder().encoder(new JacksonEncoder()).decoder(new JacksonDecoder())
+        .logger(new Slf4jLogger()).logLevel(Level.FULL)
```

Comment:
with this config, are all request details dumped to the log? headers included?

Reply:
yes

Reply:
ok, it can be a security issue, for example we should never log the jwt we generated.
I guess that setting the level to `WARN` in the configs should be enough to prevent it, but if this is not the case it should be fixed

Reply:
maybe for the first timewe have included it because now in the test phase it might be useful to know everything that happens, maybe when it is confirmed we will switch it to a lower level

Reply:
switching to WARN no longer logs the request data, tell me what you prefer

Reply:
which is the correct behaviour.
We cannot delegate an external library to log possible sensitive data.

If you need to log something related to the request, you should write a specific log, including only data that you know it is not sensitive